pages/am/3.x/breaking-changes/breaking-changes-3.4.adoc

```diff
@@ -0,0 +1,20 @@
+= Breaking changes in 3.4
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_breaking_changes_3.4.html
+:page-folder: am/installation-guide
+:page-layout: am
+
+NOTE: For all these breaking changes the Migration Guide can be found link:/am/current/am_installguide_migration.html[here].
+
+== Gateway Cookie Session
+
+Since the beginning, AM Gateway uses a good old server side session to back up some context information during user authentication flow (login, passwordless, multi factors, consents, ...).
```

Comment:
Since the beginning, AM Gateway has been using a server side session to back up some context information during user authentication flow (such as login, passwordless, multi-factor, consent).